server/test/unit/server/webservices/test_application.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from mock import patch
+
+from pulp.server.webservices.application import wsgi_application
+from pulp.server.initialization import InitializationException
+
+
+class TestApplication(unittest.TestCase):
+
+    @patch('pulp.server.webservices.application._initialize_pulp')
+    def test_wsgi_application_exception(self, mock_initialize_pulp):
+        mock_initialize_pulp.side_effect = Exception
```

Comment:
L#13 is a class type and L#18 is a specific instantiated exception. Newer versions of mock use one or the other, but I think older ones expected a instance not a class. It's probably good to have side_effect be set to an instance. Not a required change, but more of a style thing.